src/pages/patientView/timeline2/TimelineWrapper.tsx

```diff
@@ -344,7 +366,7 @@ const TimelineWrapper: React.FunctionComponent<ITimeline2Props> = observer(
                             { re: /improving/i, color: 'rgb(44, 160, 44)' },
                             { re: /worsening/i, color: 'rgb(214, 39, 40)' },
                         ];
-                        cat.items.forEach((event, i) => {
+                        function getEventColor(event: TimelineEvent) {
```

Comment:
lets just pull this out since it's used again below.  and pass in the string constant